phalcon/Mvc/CollectionInterface.zep

```diff
@@ -124,9 +124,4 @@ interface CollectionInterface
      * @param mixed id
      */
     public function setId(id);
-
-    /**
-     * Check whether validation process has generated any messages
-     */
-    public function validationHasFailed() -> bool;
```

Comment:
Why you decided to remove this method? I would like to see a descriptive reason in the change log

Reply:
The method did not exist in the collection, I just synchronized